packages/server/server/src/helpers/mock.ts

```diff
@@ -67,64 +76,93 @@ export const registerMockHandlers = async ({
   }
 
   if (!mockFilePath) {
-    return null;
+    return undefined;
   }
 
-  const { default: mockModule, config } = await import(mockFilePath);
+  const { default: mockHandlers, config } = (await import(
+    mockFilePath
+  )) as MockModule;
 
   const enable = config?.enable as
     | boolean
     | ((req: IncomingMessage, res: ServerResponse) => boolean)
     | undefined;
 
   if (enable === false) {
-    // eslint-disable-next-line consistent-return
-    return;
+    return undefined;
   }
 
-  if (!mockModule) {
+  if (!mockHandlers) {
     throw new Error(`Mock file ${mockFilePath} parsed failed!`);
   }
 
-  Object.entries(mockModule as MockConfig).forEach(([key, handler]) => {
+  return {
+    default: mockHandlers,
```

Comment:
Maybe `mockHandlers` is better than `default`?

Reply:
Yes, Great idea

Reply:
Done～